### PR TITLE
[PC-10209] handle non error sentry

### PR DESCRIPTION
**Pull request by marionnousvalentin**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-10209

## Checklist

I have:

- [ ] Made sure the title of my PR follows the convention `[PC-10209] <summary>`.
- [ ] Made sure my feature is working on the relevant real / virtual devices.
- [ ] Written **unit tests** for my feature.
- [ ] Written **documentation on Notion** for my feature.
- [ ] Added new reusable components to the AppComponents page and communicate to the team on slack
- [ ] Added a **screenshot** for UI tickets.
- [ ] Attached a **ticket number** for any added TODO/FIXME \
       (for tech tasks, give the best context about the TODO resolution: what/who/when).

## Screenshots

| \*iOS - [Phone name] | \*Android - [Phone name] | Tablet - [Phone name] | Little - [Phone name] |
| -------------------- | :----------------------: | --------------------: | --------------------: |
|                      |                          |                       |                       |

## Deploy hard

If native code (ios/android) was modified, **after** the PR is merged, on the master branch, upgrade the **app version** (+1 patch):

- if you want an hard deployment of the testing environment, use `yarn trigger:testing:deploy`

[PC-10209]: https://passculture.atlassian.net/browse/PC-10209

### CODE REVIEW

File: src/features/errors/pages/AsyncErrorBoundary.tsx [hunk 1]
```diff
@@ -6,8 +6,8 @@ import { useQueryErrorResetBoundary } from 'react-query'
 import styled from 'styled-components/native'
 
 import { UseNavigationType } from 'features/navigation/RootNavigator'
-import { AsyncError, MonitoringError } from 'libs/errorMonitoring'
-import { errorMonitoring } from 'libs/errorMonitoring'
+import { AsyncError, MonitoringError } from 'libs/monitoring'
+import { eventMonitoring } from 'libs/monitoring'
```
Comment (antoinewg) [inline src/features/errors/pages/AsyncErrorBoundary.tsx:10]: tu peux les factoriser

File: src/libs/idCheck/IdCheckContextProvider.tsx [hunk 2]
```diff
@@ -19,7 +19,7 @@ export const IdCheckContextProvider = memo(function IdCheckContextProvider({
       dsmUrl={env.DSM_URL}
       personalDataDocUrl={env.DOC_PERSONAL_DATA_URL}
       cguDocUrl={env.DOC_CGU_URL}
-      errorMonitoring={errorMonitoring}
+      errorMonitoring={eventMonitoring}
```
Comment (antoinewg) [inline src/libs/idCheck/IdCheckContextProvider.tsx:22]:
```suggestion
      eventMonitoring={eventMonitoring}
```

mais pas nécessaire

File: src/libs/monitoring/__tests__/messages.test.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+import { eventMonitoring } from 'libs/monitoring'
+
+import { MonitoringMessage } from '../messages'
+
+describe('MonitoringMessage', () => {
+  const message = 'message'
+  it('should call eventMonitoring.captureMessage() on new MonitoringMessage instance with default info level', () => {
+    new MonitoringMessage(message)
+    expect(eventMonitoring.captureMessage).toBeCalledWith(message, { level: 'info' })
```
Comment (antoinewg) [inline src/libs/monitoring/__tests__/messages.test.ts:11]: y'a pas l'équivalent dans Severity ?

**Review by antoinewg — APPROVED** [review]

File: src/libs/idCheck/IdCheckContextProvider.tsx (hunk 2 above)
Comment (marionnousvalentin) [reply src/libs/idCheck/IdCheckContextProvider.tsx:22]: ça vient de idCheck :(

File: src/features/errors/pages/AsyncErrorBoundary.tsx (hunk 1 above)
Comment (marionnousvalentin) [reply src/features/errors/pages/AsyncErrorBoundary.tsx:10]: bien vu, je maj

File: src/libs/monitoring/__tests__/messages.test.ts (hunk 3 above)
Comment (marionnousvalentin) [reply src/libs/monitoring/__tests__/messages.test.ts:11]: bien vu, je vais remplaçer 'info'

**Pull request merged**